.travis.yml

```diff
@@ -0,0 +1,38 @@
+language: objective-c
+
+install:
+  - gem install xcpretty
+
+env:
+  global:
+    - LC_CTYPE=en_US.UTF-8
+    - LANG=en_US.UTF-8
+    - FRAMEWORK_NAME="ReSwiftRouter"
+    - UPDATE_DOCS="false"
+
+matrix:
```

Comment:
Hi @Ben-G,

The `osx_image`s can be hoisted to the top level as they are repeated in each case such as:

```
language: objective-c
osx_image: xcode8.2
[...snip...]
matrix:
  include:
    - env: SCHEME="macOS"  SDK="macosx10.12"           DESTINATION="arch=x86_64"
    - env: SCHEME="iOS"    SDK="iphonesimulator10.2"   DESTINATION="OS=10.1,name=iPhone 6S Plus"
    - env: SCHEME="tvOS"   SDK="appletvsimulator10.1"  DESTINATION="OS=10.1,name=Apple TV 1080p"
```

They can always be put back in the includes later if we desire to test different Xcode versions in the same test suite.